src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+
+import duke.time.DateTime;
+
+public class Deadline extends Task {
+    private String by;
```

Comment:
I think that you can use more descriptive name for the field like 'dueDate' instead of using 'by'.